build.sbt

```diff
@@ -16,21 +16,7 @@ val jsSettings = Def.settings(
 )
 
 val nativeSettings = Def.settings(
-  libraryDependencies += "org.scala-native" %%% "junit-runtime" % nativeVersion % Test,
-  addCompilerPlugin("org.scala-native" % "junit-plugin" % nativeVersion cross CrossVersion.full),
-  pomPostProcess := { node =>
-    import scala.xml._
-    import scala.xml.transform._
-    new RuleTransformer(new RewriteRule{
-      override def transform(n: Node) =
-        if (n.label == "dependency" && (n \ "artifactId").text.startsWith("junit-runtime_native"))
-          NodeSeq.Empty
-        else
-          n
-    }).transform(node)(0)
-  },
-  testOptions += Tests.Argument(TestFrameworks.JUnit, "-a", "-s", "-v"),
-  mimaPreviousArtifacts := Set.empty,
+  tlVersionIntroduced := Map("3" -> "3.1.0")
```

Comment:
In general, you should never set `mimaPreviousArtifacts := Set.empty`, this is dangerous to forget :) Either
- you are not publishing that module so `.enablePlugins(NoPublishPlugin)`
- you have not published it yet, so `tlVersionIntroduced := Map("3" -> "x.y.z")`